netutils/acl.py

```diff
@@ -179,55 +207,80 @@ def __init__(self, data: t.Any, *args: t.Any, **kwargs: t.Any):  # pylint: disab
         Examples:
             >>> from netutils.acl import ACLRule
             >>>
-            >>> acl_data = dict(
+            >>> rule = ACLRule(
```

Comment:
I think we need a loader of sorts, something where you can just through it a dictionary, and it handles everything else.

Reply:
This is now supported :
```
acl_data = {'action': permit, src='', dst='' ... }
ACLRule(**acl_data)
```
Can't think of simpler way ?